oUnit.opam

```diff
@@ -7,18 +7,14 @@ dev-repo: "git+https://github.com/gildor478/ounit.git"
 doc: ["http://ounit.forge.ocamlcore.org/api-ounit/index.html"]
```

Comment:
Keep the name of the file to be ounit.opam. This is how it's called in opam and it's case sensitive.